en/config-sentinel.html

```diff
@@ -109,3 +107,64 @@ <h2 id="user-interface">User interface</h2>
 Use <a href="reference/vespa-cmdline-tools.html#vespa-model-inspect">
     vespa-model-inspect host <em>hostname</em></a> to list services running on a node.
 </p>
+
+
+
+<h2 id="cluster-startup">Cluster startup</h2>
+<p>
+  One can configure Vespa to do a <em>Safe Cluster Startup</em>,
+  setting a minimum node available ratio.
+  Use this to control the Vespa cluster start sequence.
+  The config sentinel will not start services on a node unless it has connectivity to a minimum other nodes.
+</p>
+<p>
+  The safe mode is disabled by default,
+  enable it by setting configuration on top level in <em>services.xml</em> (since Vespa-7.415):
+</p>
+<pre>
+&lt;services&gt;
+    &lt;config name="cloud.config.sentinel"&gt;
+        &lt;connectivity&gt;
+            &lt;maxBadOutPercent&gt;60&lt;/maxBadOutPercent&gt;
```

Comment:
I'm considering changing the name of the config variables:
maxBadPercent (all problems count)
maxOneWayCount (affirmed missing connectivity one way)

Reply:
yes, those are better names!